### DISCUSSION TITLE
Provide an extension point for CP template suggestions

### DISCUSSION THREAD

**Discussion by stanislavprokopov** [craftcms/cms#10372] [Ideas] [answered]
There is a way for the plugins to provide custom template roots, for example allowing to use them when configuring Section site settings templates, but there is now way to make the CP template autocomplete (craft.cp.getTemplateSuggestions) to suggest templates from custom template roots. It would be nice to have a way to make autocomplete suggest these templates.

**Comment by brandonkelly** [comment, marked as answer]
Custom template roots definitely should have been considered by default. Just made that happen for the next release (fe25e04a05fdf2c3c77cb66cb265f25aead0681f).

**Reply by brandonkelly** [reply]
Craft 3.7.29 is out now with that change. Thanks for the suggestion!

**Reply by stanislavprokopov** [reply]
Thanks! That was quick :D
